src/hardware/driver/mod.rs

```diff
@@ -51,7 +50,7 @@ impl ChannelVariant {
     const R_OUT_LN: f32 = 40.0; // Low noise side output resistor
     const R_OUT_HP: f32 = 0.068; // High power side output resistor
```

Comment:
Clear names, e.g. "R_SHUNT_LN". And always document units.
What does the "OUT" mean?